module/VuFind/src/VuFind/View/Helper/Root/SearchTabs.php

```diff
@@ -246,30 +247,37 @@ public function getCurrentHiddenFilterParams(
     ) {
         if (!isset($this->cachedHiddenFilterParams[$searchClassId])) {
             $view = $this->getView();
-            $searchTabs = $view->plugin('searchTabs');
-            $hiddenFilters = $searchTabs->getHiddenFilters(
+            $hiddenFilters = $this->getHiddenFilters(
```

Comment:
Well, that's just bizarre. Why did we pull this out of the view manager instead of just using $this? Glad you noticed and fixed that...

Reply:
No idea, but it was probably my fault anyway. :)